enhancements/monitoring/metrics-collection-profiles.md

```diff
@@ -0,0 +1,430 @@
+---
+title: metrics-collection-profiles
+authors:
+  - @JoaoBraveCoding
+reviewers:
+  - @openshift/openshift-team-monitoring
+approvers:
+  - TBD
+api-approvers: "None"
+creation-date: 2022-12-06
+last-updated: 2023-07-19
+tracking-link:
+  - https://issues.redhat.com/browse/MON-2483
+  - https://issues.redhat.com/browse/MON-3043
+---
+
+# Metrics collection profiles
+
+## Summary
+
+The core OpenShift components ship a large number of metrics. A 4.12-nightly
+cluster on AWS (3 control plane nodes + 3 worker nodes) currently produces
+around 350,000 unique timeseries, and adding optional operators increases that
+number. Users have repeatedly asked for a supported method of making Prometheus
+consume less memory and CPU, either by increasing the scraping interval or by
+scraping fewer targets.
+
+These modifications are currently not possible, because the OpenShift operators
+manage their service/pod monitors (either directly or via the cluster version
+operator) and any manual modification would be reverted. This proposal outlines
+a solution for allowing users to control the amount of data being collected
+which can meet their needs.
+
+## Motivation
+
+OpenShift is an opinionated platform, and the default set of metrics has of
+course been crafted to match what we think the majority of users might need.
+Nevertheless, users have repeatedly asked for the ability to reduce the amount
+of memory consumed by Prometheus either by lowering the Prometheus scrape
+intervals or by modifying ServiceMonitors.
+
+Users currently can not control the ServiceMonitors scraped by Prometheus since
+some of the metrics collected are essential for other parts of the system to
+function properly: alerting rules, console dashboards, resource metrics API,
+horizontal/vertical pod autoscaling and Red Hat Telemetry. Users also are not
+allowed to tune the interval at which Prometheus scrapes targets as this again
+can have unforeseen results that can hinder the platform: a low scrape interval
+value may overwhelm the platform Prometheus instance while a high interval value
+may render some of the default alerts ineffective.
+
+The goal of this proposal is to allow users to pick their desired level of
+scraping while limiting the impact this might have on the platform, via
+resources under the control of the cluster-monitoring-operator and other
+platform operators.
```

Comment:
Since we now only support two profiles and it's out of the goal of adding further profiles I feel like this is not relevant anymore. Since if a developer does not adhere to this feature their operator will just default to exporting all metrics.